page/src/components/EventCount/EventCount.jsx

```diff
@@ -0,0 +1,14 @@
+const getCountText = (count) => {
+    const getCountText = (count) => {
```

Comment:
Is it normal these two lines `const getCountText = (count) => {`?

Reply:
nope, I'm testing GitHub codespace, must I messed up something. If you want to test it, you need to serve the app with to avoir [502 Bad Gateway error](https://github.com/orgs/community/discussions/28563#discussioncomment-4880737):
```
npm run dev -- --host 0.0.0.0
```